seek_to.go

```diff
@@ -0,0 +1,18 @@
+package levigo
+
+// #include "seek_to.h"
+import "C"
+import "unsafe"
+
+type SeekResult struct {
+	Valid bool
+	Equal bool
```

Comment:
It'd be nice to get the value in here eventually.  Cut the cgo overhead even more.